src/View/SSViewer.php

```diff
@@ -332,6 +332,8 @@ public static function hasTemplate($templates)
      * @var boolean
      */
```

Comment:
I think we should have a similar property for `set_themes`; `$current_themes`?

Reply:
yeah, I remember the documentation recommended setting theme in the `_config.php` files, so making sure the recommendation uses `SSViewer::set_themes()` along with the `$current_themes` static property would be a good idea :)